### DISCUSSION TITLE
Select 2 Issue with livewire due to wire:ignore?

### DISCUSSION THREAD

**Discussion by hammaadd** [livewire/livewire#2976] [1. Help]
I am new to livewire, was trying to integrate the tagable select2 But haven't got successful yet. 
Everything works fine but when I add a new tag in the select2 after adding data into the database, New added tag has to be there in the select2 dropdown but due to wire:ignore it didn't get updated. But if I remove wire:ignore the select2 initialization not work.

**Edit-note.blade.php ( View file )** 
```
<div class="input--field">
                    <label for="tags2">Tags</label>
                    <div wire:ignore wire:key="tags-drop">
                        <select class="multiple-select" id="tags2" multiple="multiple" name="tags">
                            @forelse($tagsG as $tg)
                            <option value="{{$tg->id}}">{{$tg->tag}}</option>
                            @empty
                            No tag added
                            @endforelse
                        </select>
                    </div>
                </div>
@push('script_s')
<script>
    $(document).ready(function() {
        $('#tags2').select2({
            tags:true
        });
    });

    window.livewire.on('update-tags-ev', message => {
        $('#tags2').select2({
            tags:true
        });
        $('#tags2').val(@this.get('tags')).trigger('change');
        
})
document.addEventListener('livewire:load', function () {
            $('#tags2').select2({
                tags:true
            }); 
        $('#tags2').on('select2:select', (e) => {
            @this.emit('setTags2', $('#tags2').select2('val'));
        });

        $('#tags2').on('select2:unselect', (e) => {
            @this.emit('setTags2',$('#tags2').select2('val'));
        });

        $('#tags2').val(@this.get('tags')).trigger('change');

    </script>


@endpush
```

**EditNote.blade.php ( Livewire File)**

```
public function render()
    {
        $this->tagsG = Tag::where('user_id',Auth::id())->where('status','active')->get();
        $this->wrkspcs = Workspace::where('created_by',Auth::id())->where('status','active')->get();
        return view('livewire.edit-note');
    }
```

**Comment by drbornot** [comment]
Use wire:ignore.self

**Reply by Pulstarter** [reply]
Hi!

I have made an implementation of selec2 with Livewire (2.5.1) and AlpineJS (3.0.6)

[livewire-select2](https://github.com/Pulstarter/livewire-select2/blob/8eb6fc2b779c02ed174219125d06b481737a8e73/select2.blade.php)

Basically you need to treat everything with select2 events, passing a wire:model with your Livewire value and an $options array with your select2 options. This $options array can be updated from your server and select2 will catch the changes fine.

Usage:
    <x-select2
                wire:model.debounce.300ms="<LIVEWIRE_MODEL>"                
                label="{{__('LABEL')}}"
                options="<LIVEWIRE_OPTIONS_ARRAYNAME>"
     />

Hope it helps.
